_data-prepper/pipelines/configuration/processors/trace-peer-forwarder.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Trace peer forwarder
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Trace peer forwarder
+
+The `trace peer forwarder` processor is used to reduce the number of events that are forwarded in a trace analytics pipeline by half when using [Peer forwarder]({{site.url}}{{site.baseurl}}/data-prepper/managing-data-prepper/peer-forwarder/). It groups the events based on `trace_id` similar to the `service_map_stateful` and `otel_trace_raw ` processors. 
+
+In [Trace analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/), each event is duplicated when it is sent from the `otel-trace-pipeline` to the `raw-pipeline` and to the `service-map-pipeline`. The event is forwarded once in each pipeline. When you use this processor, the event is forwarded once in the `otel-trace-pipeline` pipeline to the correct peer. 
+
+## Usage
+
+To get started with `trace peer forwarder`, create the following `pipeline.yaml` file along with [Peer forwarder]({{site.url}}{{site.baseurl}}/managing-data-prepper/peer-forwarder/) in your `data-prepper-config.yaml` file. For more detailed information, see [Configuring Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/getting-started/#2-configuring-data-prepper).
```

Comment:
"... along with [Peer forwarder]({{site.url}}{{site.baseurl}}/managing-data-prepper/peer-forwarder/) in your `data-prepper-config.yaml` file."
This confuses me. Do you do this at the same time you create the pipeline.yaml file? Or is it already there, and you add the pipeline.yaml file in the same location as the data-prepper-config.yaml file?

Reply:
@cwillum I think it is a single step. If we re-word to "create the following `pipeline.yaml` file by using Peer forwarder in your `data-prepper-config.yaml` file." it makes more sense. I'm basing this off of https://opensearch.org/docs/latest/data-prepper/getting-started/#2-configuring-data-prepper.

@asifsmohammed Can you clarify/confirm?

Reply:
From David: these are two different steps. 
Reword the along with portion to "Using the Trace peer forwarder requires that you configure the peer forwarder."
After that, you update your pipeline to include the trace peer forwarder.


1. Configure Peer forwarder.
2. Include the trace peer forwarder in your entry pipeline.